user-interface/1005-VEST-vesting.md

```diff
@@ -1 +1,75 @@
-# Vesting
\ No newline at end of file
+# Vesting
+
+Some governance tokens may be held by a Vesting contract. This means that can be "owned" by an Ethereum key but not freely transferred until a vesting terms are complete.
+
+## list of tranches
+
+When looking to understand to overall vesting schedule for tokens, I...
+
+- **must** see a list of tranches <a name="1005-VEST-001" href="#1005-VEST-001">1005-VEST-001</a>
+- **should** see a visualization of the vesting schedule with a break down of the type of token holders (e.g. team, investors, community) <a name="1005-VEST-002" href="#1005-VEST-002">1005-VEST-002</a>
+
+For each tranche:
+
+- **must** see a tranche number <a name="1005-VEST-003" href="#1005-VEST-003">1005-VEST-003</a>
+- **could** see any annotation of what this tranche is about (e.g. community schedule A)
+- **must** see a sum of tokens in the tranche <a name="1005-VEST-005" href="#1005-VEST-005">1005-VEST-005</a>
+  - **must** see how many tokens in the tranche are locked <a name="1005-VEST-006" href="#1005-VEST-006">1005-VEST-006</a>
+  - **must** see how how many tokens in the tranche are redeemable <a name="1005-VEST-007" href="#1005-VEST-007">1005-VEST-007</a>
+- **must** see the vesting terms for each tranche (when unlocking stats and ends) <a name="1005-VEST-008" href="#1005-VEST-008">1005-VEST-008</a>
+
+... so I can understand how circulating supply could change over time.
+
+## Details of a tranche
+
+When looking into a specific tranche, I...
+
+- **must** see all the same details as the [list of tranches](#details-of-a-tranche) <a name="1005-VEST-012" href="#1005-VEST-012">1005-VEST-012</a>
+- **must** see a list of ethereum wallets with tokens in this tranche <a name="1005-VEST-013" href="#1005-VEST-013">1005-VEST-013</a>
+
+for each ethereum wallet:
+
+- **must** see the full eth address of the wallet <a name="1005-VEST-014" href="#1005-VEST-014">1005-VEST-014</a>
+- **must** see the total tokens this address holds in this tranche <a name="1005-VEST-015" href="#1005-VEST-015">1005-VEST-015</a>
+  - **must** see how many tokens in the tranche are locked <a name="1005-VEST-016" href="#1005-VEST-016">1005-VEST-016</a>
+  - **must** see how how many tokens in the tranche are redeemable <a name="1005-VEST-017" href="#1005-VEST-017">1005-VEST-017</a>
+
+... so I can see the details of how tokens are distributed in this tranche
+
+## See summary for a given Ethereum key
+
+When looking to see how many tokens I have in total, and how many I might be able to redeem, I...
+
+- **must** be able to [Connect and ethereum wallet](0004-EWAL-connect_ethereum_wallet.md) <a name="1005-VEST-018" href="#1005-VEST-018">1005-VEST-018</a>
+- **should** be able input an ethereum address
+
+for the a given Ethereum wallet/address/key:
+
+- **must** see a total of tokens across all tranches <a name="1005-VEST-020" href="#1005-VEST-020">1005-VEST-020</a>
```

Comment:
nothing in here about tranche 0 tokens

Reply:
agreed on call.
Should show if you have tokens in tranche 0 when looking at redeem

Reply:
https://github.com/vegaprotocol/specs/pull/1301/commits/d0ca0f8dca65d77a7ade2c52c67541554baf73d6